packages/core/admin/admin/src/content-manager/components/BlocksInput/Blocks/List.tsx

```diff
@@ -85,31 +102,72 @@ const handleBackspaceKeyOnList = (editor: Editor, event: React.KeyboardEvent<HTM
     currentList.children.length === 1 &&
     isText(currentListItem.children[0]) &&
     currentListItem.children[0].text === '';
-  const isNodeStart = Editor.isStart(editor, editor.selection.anchor, currentListItemPath);
+  const isListItemEmpty =
+    currentListItem.children.length === 1 &&
+    isText(currentListItem.children[0]) &&
+    currentListItem.children[0].text === '';
   const isFocusAtTheBeginningOfAChild =
-    editor.selection.focus.offset === 0 && editor.selection.focus.path.at(-1) === 0;
+    editor.selection.focus.offset === 0 && editor.selection.focus.path.at(-2) === 0;
 
   if (isListEmpty) {
-    event.preventDefault();
-    replaceListWithEmptyBlock(editor, currentListPath);
-  } else if (isNodeStart) {
-    Transforms.liftNodes(editor, {
-      match: (node) => !Editor.isEditor(node) && node.type === 'list-item',
+    const parentListEntry = Editor.above(editor, {
+      at: currentListPath,
+      match: (node) => !Editor.isEditor(node) && node.type === 'list',
     });
-    // Transforms the list item into a paragraph
-    Transforms.setNodes(
-      editor,
-      { type: 'paragraph' },
-      {
-        hanging: true,
-      }
-    );
+    if (!parentListEntry) {
+      event.preventDefault();
+      replaceListWithEmptyBlock(editor, currentListPath);
+    }
   } else if (isFocusAtTheBeginningOfAChild) {
+    // If the focus is at the beginning of a child node we need to replace it with a paragraph
     Transforms.liftNodes(editor, {
       match: (node) => !Editor.isEditor(node) && node.type === 'list-item',
     });
-    // If the focus is at the beginning of a child node we need to replace it with a paragraph
     Transforms.setNodes(editor, { type: 'paragraph' });
+  } else if (isListItemEmpty) {
+    const previousEntry = Editor.previous(editor, {
+      at: currentListItemPath,
+    });
+
+    const nextEntry = Editor.next(editor, {
+      at: currentListItemPath,
+    });
+
+    if (previousEntry && nextEntry) {
+      const [previousNode] = previousEntry;
+      const [nextNode] = nextEntry;
+      event.preventDefault();
```

Comment:
You've added this prevent default in two places in this PR that aren't very close to the top of the event, how come they're needed so far down?